UPGRADE-6.3.md

```diff
@@ -103,3 +103,80 @@ None.
 ## Database Fields
 
 None.
+
+## Backend Theme Library
```

Comment:
We should a notice for the changed jQuery version, too (even if the version might change again until release).

Reply:
This PR ist totally immediate. I already added this info in the follow up PR.